### PR TITLE
US6285 - Provide sign-in form example

### PR DESCRIPTION
**Pull request by nogorilla**
Have a matching [PR](https://github.com/crdschurch/crds-styles/pull/31) on `crds-styles`

### CODE REVIEW

File: src/app/ui-components/sign-in/sign-in.component.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<div class="jumbotron">
+  <div class="container">
+    <content-block id="UIAlertsSectionHeader"></content-block>
```
Comment (tcmacdonald) [inline src/app/ui-components/sign-in/sign-in.component.html]: Need to update this to support a custom header for this section. Please also submit a corresponding PR to the ss-cms repo for this purpose.

File: src/app/ui-components/sign-in/sign-in.component.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<div class="container">
+  <div class="page-header">
+    <h2>Sign in Example</h2>
+  </div>
+
+  <div class="crds-example">
+    <header class="page-header">
+      <h2>Sign In</h2>
+    </header>
+    <form action="/" novalidate>
```
Comment (tcmacdonald) [inline src/app/ui-components/sign-in/sign-in.component.html]: Does this allow the form to actually submit to root? We do not want that sort of behavior.

File: src/app/ui-components/sign-in/sign-in.component.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <content-block id="UIAlertsSectionHeader"></content-block>
+  </div>
+</div>
+
+<div class="container">
+  <div class="page-header">
+    <h2>Sign in Example</h2>
+  </div>
+
+  <div class="crds-example">
```
Comment (tcmacdonald) [inline src/app/ui-components/sign-in/sign-in.component.html]: I also feel like we need some copy explaining what this pattern is, why it exists, how one might implement the forms / validation within NG1 and/or NG2 (we don't need to a lot of implementation detail but should link over to the relevant documentation).

File: src/app/ui-components/sign-in/sign-in.component.html (hunk 2 above)
Comment (nogorilla) [inline src/app/ui-components/sign-in/sign-in.component.html]: it doesn't but I'll take it out anyways

**Comment by nogorilla** [comment]
@tcmacdonald Updated per your review. Sibling PR for [`SS-CMS`](https://github.com/crdschurch/SS-CMS/pull/249)

**Comment by nogorilla** [comment]
@tcmacdonald Requested changes have been made

File: src/app/ui-components/sign-in/sign-in.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SignInComponent
+      ],
+      imports: [
+        ContentBlockModule,
+        HttpModule,
+        RouterTestingModule.withRoutes([])
```
Comment (tcmacdonald) [inline src/app/ui-components/sign-in/sign-in.component.spec.ts]: These imports are not necessary.

**Comment by nogorilla** [comment]
@tcmacdonald updates have been made and ready for review

**Pull request merged**